### ISSUE TITLE
[BUG] Symbol’s value as variable is void: +irc--workspace-name

### ISSUE THREAD

**Issue by flyingns** [hlissner/doom-emacs#1616]
When I:

`M-x circe`

It reported a message:

`Symbol’s value as variable is void: +irc--workspace-name`

**Comment by hlissner** [comment]
Are you on the `develop` or `straight` branch?

**Comment by flyingns** [comment]
develop

**Comment by hlissner** [comment]
This has been fixed on the straight branch in 2b1c60f. It will be around a week until that branch is merged, so use this workaround for the time being:

```elisp
;; in ~/.doom.d/config.el
(defvar +irc--workspace-name "*IRC*")
```

**Comment by flyingns** [comment]
reported：

`persp-contain-buffer-p: Wrong type argument: perspective, :nil `
